README.md

```diff
@@ -2,7 +2,7 @@
 
 ### License
 
-Copyright 2015-2021 SonarSource.
+Copyright 2015-2022 SonarSource.
```

Comment:
This prompted me to look at the copyright notices in the other files, and they are all out of date. I've created a separate PR for them.